docs/usage/Snapshot.md

```diff
@@ -0,0 +1,27 @@
+## Enabling support for Snapshots with CAM
+When using cloud-provider snapshots with CAM, it is important to ensure that
+all of the necessary preconditions are met. This document contains any
+provider-specific configuration information required to get snapshot support.
+
+### Prerequisites
+The CAM web console must contain the following:
+* Source cluster
+* Target cluster, which is added automatically during the CAM tool installation
+* Both source and target cluster must be running on the same cloud provider and in the same region
+  * For Azure clusters, ensure that the Resource Group is the same for both the source and target cluster. For more information see step #4 [here](https://github.com/fusor/mig-operator/blob/master/docs/usage/Cluster.md#procedure)
```

Comment:
"ensure that the Resource Group is the same for both the source and target cluster" -> resource group needs to be unique per cluster rather than the same. Needs to be the unique resource group that the cluster was provisioned into.